### PR TITLE
feat(FEC-7808): added the option to disable cap level on fps drop

### PR DESCRIPTION
**Pull request by odedhutzler**
added the option to disable cap level on fps drop
and added new abr configuration:
```
abr: {
 fpsDroppedMonitoringThreshold: number,
 fpsDroppedFramesInterval: number,
  capLevelOnFPSDrop: boolean
}
```

### Description of the Changes

Please add a detailed description of the change, weather it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -77,6 +76,10 @@ var config = {
         format: 'progressive'
       }
     ]
+  },
+  abr: {
+    fpsDroppedFramesInterval: 5000,
+    fpsDroppedMonitoringThreshold: 0.2
```
Comment (OrenMe) [inline docs/configuration.md]: missing `capLevelOnFPSDrop`

File: src/engines/dropped-frames-watcher.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ class DroppedFramesWatcher extends FakeEventTarget {
       this._eventManager.listen(this._mediaSourceAdapter, CustomEventType.FPS_DROP, event => this._triggerFPSDrop(event.payload.data));
       return;
     }
-    if (this._getDroppedAndDecodedFrames()[0] === NOT_SUPPORTED) {
-      DroppedFramesWatcher._logger.debug('Dropped frame watcher is not supported on this environment');
+    if (this._getDroppedAndDecodedFrames()[0] === NOT_SUPPORTED || !this._config.capLevelOnFPSDrop) {
```
Comment (OrenMe) [inline src/engines/dropped-frames-watcher.js]: switch order - config is more important than API support.
Also, the debug level is misleading, flag means it is disabled, not supported check should mention that it is not supported.

File: docs/configuration.md (hunk 1 above)
Comment (odedhutzler) [reply docs/configuration.md]: fixed

File: src/engines/dropped-frames-watcher.js (hunk 2 above)
Comment (odedhutzler) [reply src/engines/dropped-frames-watcher.js]: changed

**Pull request merged**